com.onlyoffice.DesktopEditors.json

```diff
@@ -0,0 +1,93 @@
+{
+  "app-id": "com.onlyoffice.DesktopEditors",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "19.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "/app/desktopeditors/onlyoffice-desktopeditors.sh",
+  "separate-locales": false,
+  "tags": [
+    "AGPL-3.0"
+  ],
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--device=all",
+        "--filesystem=xdg-videos:ro",
+        "--filesystem=xdg-pictures:ro",
+        "--filesystem=xdg-download",
+        "--filesystem=host",
+        "--talk-name=org.gtk.vfs.*"
+   ],
+  "modules": [
+    {
+      "name": "onlyoffice",
+      "buildsystem": "simple",
+      "build-commands": [
+        "mv desktopeditors /app/desktopeditors",
+        "chmod +x /app/desktopeditors/DesktopEditors",
+        "chmod +x /app/desktopeditors/onlyoffice-desktopeditors.sh",
+        "chmod +x /app/desktopeditors/post_install.sh",
+        "/app/desktopeditors/post_install.sh",
+        "install -d /app/share/applications",
+        "install -Dm644 /app/desktopeditors/asc-de-64.png /app/share/icons/hicolor/256x256/apps/${FLATPAK_ID}.png",
+        "install -Dm644 com.onlyoffice.DesktopEditors.appdata.xml /app/share/appdata/com.onlyoffice.DesktopEditors.appdata.xml"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download.onlyoffice.com/install/desktop/editors/linux/old/onlyoffice-desktopeditors-x64.tar.gz",
```

Comment:
Make sure you double-check that they allow redistribution of their binaries (just because it's open source doesn't mean they do BTW).

Reply:
I don't think we'll be in trouble considering the Desktop Editors are under AGPL along with the entire suite so it won't be a problem.

Reply:
@agolybev @maxkadushkin?

Reply:
Yes, Desktop Editors is under agpl  v3.0 , but this package contains old version of Desktop Editors.